packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,254 @@
+- [ViD Select Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=2597-83231&mode=design)
+
+## Usage
+When to use:
+- To present a pre-defined list of options
+- For filtering or sorting data in tables or lists
+- For selecting settings or preferences
+- For single selection
+
+When not to use:
+- To allow end-users to add custom options
+- When there are fewer than three options
+- For complex and/or hierarchical data
+- For multi selection
+
+### Anatomy
+![ ](spec-images/select-1.png)
+
+| Element | Description |
+| ---------- | ---------- |
+| Label | Text that tells the end-user what to expect in the list of options | 
+| Helper text | Text that provides extra context or instruction about what the end-user should select | 
+| Field button | Click area that opens the menu and displays the current selected option| 
+| Menu | A list of options to choose from displayed as an *open* state | 
+| Menu item | A selection the end-user can make, shown with other options in the menu | 
+
+#### Label
+Selects should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases).
+
+Keep the label text as concise as possible, limited to a single line of text. Use sentence casing.
+
+In a form, use the label to indicate whether the select is optional or required for the end-user to interact with. If most of the components in the form are required, indicate which components are optional. If most of the components in the form are optional, indicate which components are required. Required or optional indication in the label text should always follow the pattern "[Label] (optional)" or "[Label] (required)".
+
+![ ](spec-images/select-2.png)
+
+#### Helper Text
+> NOTE: NEEDS VISUAL DESIGN FOR TYPOGRAPHY
+
+Only include helper text to provide information that aids the end-user in choosing the right selection. Keep the helper text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-3.png)
+
+#### Field Button
+Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that propts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.
+
+![ ](spec-images/select-4.png)
+
+#### Menu & Menu Items
+A list of options to select, displayed as an open state. Order menu items alphabetically by default, unless another ordering will provide value to the end-user. Keep the menu item text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-5.png)
+
+### Related Components
+To allow end-users to enter a custom or create a new menu item, use a [**combobox**](/packages/nimble-components/src/combobox/specs/IxD.md).
```

Comment:
Sorry, I don't understand this comment. The functional difference between the select and the combo box is that the end-user can type to enter and commit a custom value in a combo box. I can remove the language "create a new menu item", if we don't plan on implementing that as a feature of the combo box (i.e. the user can enter a custom value, but that custom value will never be added as a menu item). But you don't type to enter a custom value on a select, because that's what a combo box is for.

Reply:
I'm seeing this feature less as a "what a user can use as a value", and more of a "allow user to add new items to dropdown". I think it's odd to provide guidance to a user that if they were originally considering using a `Select`, which could have been motivated by them needing to use the `value/ngValue` APIs on the `list-option`, to use a `Combobox` if they also wanted to interactively create a new menu item (as the `Combobox` is still incompatible with their needs). There's a logical leap there that is grounded in the fact that the added menu item _must not_ have a value associated with it other than what was typed in.

Ultimately, this all could be moot if we agree that that feature should not be implemented on the `Combobox`, which I will make my case for on [its PR](https://github.com/ni/nimble/pull/1521). It's additionally moot in that this document is _not_ client-facing. I think the actual guidance we would be giving for why a user would choose a `Select` versus a `Combobox` would be fairly different.

Reply:
Sure, this is the user experience centric guidance meant as a first pass for content that will eventually (after being edited and combined with other guidance) live in Storybook. Any technical/functional guidance for choosing a component should be included in client-facing documentation, but I don't think the IxD spec is the correct place to include it.